scenario_player/runner.py

```diff
@@ -195,7 +195,11 @@ def run_scenario(self):
         )
         if self.token_address not in registered_tokens:
             code, msg = self.register_token(self.token_address, first_node)
-            if not 199 < code < 300:
+            for i in range(5):
```

Comment:
Heh, this will check the return code 5 times but not perform the request again ;)